11ty/definitions/blacklist-whitelist.md

```diff
@@ -0,0 +1,63 @@
+---
+title: blacklist whitelist
+slug: blacklist-whitelist
+flag:
+  level: avoid
+  text: racist term
+defined: true
+excerpt: 'umbrella term used to represent a list of entities/people who are to be allowed or denied access to a service, privilege, or recognition: to "blacklist" is "to exclude", to "whitelist" means "to include".'
+speech: noun; verb
+alt_words:
+  blacklist (noun):
+    - blocklist
+    - denylist
+    - excludelist
+  whitelist (noun):
+    - allowlist
+    - passlist
+    - trustlist
+    - safelist
+  to blacklist (verb):
+    - to block
+    - to deny
+    - to exclude
+  to whitelist (verb):
+    - to allow
+    - to trust
+    - to include
+reading:
+  - text: UK security agency bans blacklist and whitelist for being racist
+    href: https://www.techspot.com/news/85080-uk-cyber-security-agency-stop-using-racist-blacklist.html
+  - text: Google Chrome team moves away from black/whitelist to be more inclusing
+    href: https://9to5google.com/2020/06/12/google-android-chrome-blacklist-blocklist-more-inclusive/
+  - text: Google developers styleguide entry for blacklist
+    href: https://developers.google.com/style/word-list#blacklist
+  - text: '"Blacklists” and “whitelists": a salutary warning concerning the prevalence of racist language in discussions of predatory publishing'
+    href: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/
+---
+
+To allow or deny access to a service, a privilege, or recognition.
```

Comment:
```suggestion
colloquially: umbrella term used to represent a list of entities/people who are to be allowed or denied access to a service, privilege, or recognition: to "blacklist" is "to exclude", to "whitelist" means "to include;" carries anti-Black sentiments
```
I think what you have in the excerpt is honestly more complete. I also added the last phrase because we try to include the issues immediately so that if people only read the top level definition, they get this context.